### ISSUE TITLE
Input filename with multiple dots

### ISSUE THREAD

**Issue by pariasm** [fperezgamonal/faldoi-ipol#1]
Input filenames with more than one dot in their filename return an error.

**Comment by fperezgamonal** [comment]
Hello Pablo,

Thanks for reporting that bug. Do you mind clarifying what you mean by *multiple dots* in the filename. I suppose you mean something similar to the examples below:

```
../example_data/images_22.01.txt
~/Documents/images.flow/image_pair.txt
```
If you can add the examples you tested, I'll appreciate it. I anticipate that it may have something to do with I/O functions that were already provided in the original source code of FALDOI which are pretty unknown to me (maybe we will need to ask Enric Meinhardt @mnhrdt (since I cannot find Roberto's github user ATM)).

Cheers,

Ferran.

**Comment by pariasm** [comment]
Yes, I mean exactly examples like that. Any filename with more than one dot is incorrectly parsed.

This is an example. You can recreate it by taking any pair of images and adding a dot to their filename.
```
$ cat faldoi-tmp/files-faldoi.txt 
faldoi-tmp/a.001.png
faldoi-tmp/a.002.png
$ ./faldoi_sift.py faldoi-tmp/files-faldoi.txt -res_path ./faldoi-tmp/
Code blocks activation value:
        descriptors =   True
        matchings =     True
        sparse_flow =   True
        local_of =      True
        global_of =     True

Traceback (most recent call last):
  File "./faldoi_sift.py", line 143, in <module>
    sequence = data[0].split('.')[-2].split('/')[-2]  # not used
IndexError: list index out of range
```

The error is in the python script. It should be an easy fix.

**Comment by fperezgamonal** [comment]
It was indeed related to how I parsed the filenames in the input text file (I looked for a '.' and defined the name as everything before, which conflicted with image names with dots in them).

Now it should work as expected (reference to the [particular commit](https://github.com/fperezgamonal/faldoi-ipol/tree/51c48a08285a096203c6aa61872ce463cf291602) in case I change anything).

Cheers.
